src/styles/blocks/_card.scss

```
@@ -0,0 +1,49 @@
+.card {
+  &__item {
+    margin-bottom: 40px;
+
+    &:last-child {
+      margin-bottom: 0;
+    }
+
+    @include onTablet {
+      margin-bottom: 48px;
+    }
+    @include onDesktop {
+      margin-bottom: 0;
+    }
+  }
+
+  &__img {
+    width: 100%;
+    transition: transform 0.3s;
+
+    &:hover {
+      transform: scale(1.2);
+    }
+  }
+
+  &__name {
+    margin: 32px 0 0;
+    font-weight: 700;
+    font-size: 24px;
+    line-height: 140%;
+    color: #fff;
+  }
+
+  &__text {
+    margin-top: 8px;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 140%;
+    color: #fff;
```

create variables for all colors

```suggestion
    color: $c-white;
```